c2rust-transpile/tests/snapshots/os-specific/uint32.c

```diff
@@ -0,0 +1,2 @@
+#include <stdint.h>
+uint32_t var;
```

Comment:
How come there's a separate test for just `uint32_t` since it's covered in `types.c` already?

Reply:
I mean it was useful during development because of the order things got implemented. It's no longer necessary and is now just duplicating coverage from `types.c`; I'll remove it.